indexer/src/handlers/factory.ts

```diff
@@ -72,38 +61,22 @@ export class FactoryHandler extends BaseHandler {
             pair.token1Symbol || "Unknown"
           }`,
         },
-        "Initializing existing pair handler from database"
+        "Registering existing pair"
       );
 
-      // Create new pair handler
-      const pairHandler = new PairHandler(pairInfo);
-      await pairHandler.init(this._api);
-
-      console.log(
-        `[*] Registering pair handler ${pairHandler.constructor.name}`
-      );
-
-      // Store the handler locally for processing
-      this._pairHandlers.set(pairInfo.address, pairHandler);
+      this._pairsHandler.registerExistingPair(pair);
+      this._pairsHandler.registerPair(pairInfo);
     }
 
     this._ctx.log.info(
-      { totalPairs: this._pairHandlers.size },
-      "Successfully loaded and registered all existing pair handlers"
+      { totalPairs: existingPairs.length },
+      "Successfully loaded existing pairs into memory"
     );
   }
 
-  public async clear(): Promise<void> {
-    for (const pairHandler of this._pairHandlers.values()) {
-      await pairHandler.clear();
-    }
-  }
+  public async clear(): Promise<void> {}
 
-  public async save(): Promise<void> {
-    for (const pairHandler of this._pairHandlers.values()) {
-      await pairHandler.save();
-    }
-  }
+  public async save(): Promise<void> {}
```

Comment:
I would keep the saving pairs functionality here since it relates to the Factory handler.